provisioning-workflows/IIB/IIBv9/jcl/AUTHFS.jcl

```diff
@@ -0,0 +1,43 @@
+//S1     EXEC USSBATCH,SU=Y
+//STDOUT  DD SYSOUT=*
+//STDERR  DD SYSOUT=*
+//SYSIN   DD *
+#!/bin/sh
+# gaat recursief door directories vanaf startpunt, slaat links over
+# reageert ook goed op spaties in directory namen
+function dodir
+{
+        typeset fname savedir
+        ls | while read "fname" ; do
+                if [[ -L "$fname" ]] ; then
+                        echo 'skipped link '  "$fname"
+                elif [[ -f "$fname" ]] ; then
+                        setfacl -m group:USNZOS01:rwx "$fname"
+                        setfacl -m user:${instance-ZOSMF_INSTALL_USER}:rwx "$fname"
+
+                elif [[ -d "$fname" ]] ; then
+                        savedir=$(pwd)
+                        setfacl -m group:USNZOS01:rwx "$fname"
+                        setfacl -m user:${instance-ZOSMF_INSTALL_USER}:rwx "$fname"
+                        setfacl -m f:group:USNZOS01:rwx   ./ 
+                        setfacl -m f:user:${instance-ZOSMF_INSTALL_USER}:rwx   ./
+
+                        cd "$fname"
+                        dodir $savedir/$fname
+                        cd "$savedir"
+                fi
+        done
+        return
+}
+
+fname2=/plex/var/mqsi/brokers/${instance-IIB_BROKER_NAME_LC}/
+cd $fname2 && \
+dodir $(pwd)
+setfacl -m group:USNZOS01:rwx ./           
+setfacl -m d:group:USNZOS01:rwx   ./       
+
+setfacl -m user:${instance-ZOSMF_INSTALL_USER}:rwx ./           
+setfacl -m d:user:${instance-ZOSMF_INSTALL_USER}:rwx   ./
+
+exit 0
+
```

Comment:
Line 15, 20, 22, 36, 37  all reference hardcoded RACF groups. We should create workflow variable for that.

Reply:
I agree. Do we have to wait for Niek to make change or @RonaldKristel will you be able to make this change?